sdk/python/kubeflow/training/api/training_client.py

```diff
@@ -102,6 +102,12 @@ def train(
         model_provider_parameters=None,
         dataset_provider_parameters=None,
         trainer_parameters=None,
+        init_env_vars: Optional[
+            Union[Dict[str, str], List[Union[models.V1EnvVar, models.V1EnvVar]]]
+        ] = None,
+        worker_env_vars: Optional[
```

Comment:
Let's be consistent here for now, and use `env_vars`, similar to `create_job`.
I think for model and dataset provider, we should do this:
`model_provider_env_vars`
`dataset_provider_env_vars`

WDYT @tarekabouzeid @deepak-muley @saileshd1402 @Electronic-Waste ?